packages/client/hmi-client/src/components/stratification/tera-stratification-group-form.vue

```diff
@@ -74,7 +77,8 @@ const updatedConfig = computed<StratifyGroup>(() => ({
 	groupLabels: labels.value,
 	cartesianProduct: cartesianProduct.value,
 	directed: directed.value,
-	structure: structure.value
+	structure: structure.value,
```

Comment:
Why not removing `structure`? It is not used anywhere in this component, and you are using `useStructure` for the logic.

Reply:
So ..... my intuition is that we will need "structure" in the next iteration, in our discussion we were a bit lazy and went with the black/white route, either you have all structural combinations or no structural combinations. 

But this is kind of kicking the can down the road: model-configuration will become more difficult (resolving transitions that shouldn't be there), and simulations will be more expensive (solving relationships that result in 0s). So having a "structure" in place now makes it a little easier for the next round and the stratify-state remains compatible.